sites/all/themes/springboard/css/style.css

```diff
@@ -10069,19 +10069,39 @@ a.infographic4 p {
   .decks p.deck-summary {
     font-size: 18px;
     position: relative;
-    top: 0px;
-    left: 0px;
+    top: 0;
+    left: 0;
     width: auto;
   }
 }
 @media (min-width: 992px) {
-  /* .premade-deck-links .btn:focus, */
   .premade-deck-links .btn:hover,
   .premade-deck-links .btn.active {
     -webkit-transform: translateX(20px);
     transform: translateX(20px);
   }
 }
+@media (min-width: 450px) and (max-width: 991px) {
```

Comment:
I'd have preferred not to use min AND max widths like that, but with the buttons kind of being all over the place and certain ones being made with views/tons of nested divs, this was the quickest/easiest/cleanest way I could think to get it done.

Reply:
Agreed